bin/accessibility-automation/cypress/index.js

```diff
@@ -0,0 +1,115 @@
+        const included = includeTagArray.length === 0 || includeTags.some((include) => fullTestName.includes(include));
+        shouldScanTestForAccessibility = !excluded && included;
+      } catch (error){
+        console.log("Error while validating test case for accessibility before scanning. Error : ", error);
+      }
+    }
+    const dataForExtension = {
+      "saveResults": shouldScanTestForAccessibility,
+      "testDetails": {
+        "name": attributes.title,
+        "testRunId": '5058', // variable not consumed, shouldn't matter what we send
+        "filePath": attributes.invocationDetails.relativeFile,
+        "scopeList": [
+          attributes.invocationDetails.relativeFile,
+          attributes.title
+        ]
+      },
+      "platform": {
+        "os_name": Cypress.platform === "darwin" ? "mac" : "windows",
+        "os_version": Cypress.env("OS_VERSION"),
+        "browser_name": Cypress.browser.name,
+        "browser_version": Cypress.browser.version
+      }
+    };
+    return new Promise((resolve, reject) => {
+      if (dataForExtension.saveResults) {
+        window.parent.addEventListener('A11Y_TAP_TRANSPORTER', (event) => {
+          resolve(event.detail);
+        });
+      }
+      const e = new CustomEvent('A11Y_TEST_END', {detail: dataForExtension});
+      window.parent.dispatchEvent(e);
+      if (dataForExtension.saveResults !== true )
+        resolve();
+    });
+  }
+
+});
+
+Cypress.Commands.add('getAccessibilityResultsSummary', () => {
+  if (Cypress.env("IS_ACCESSIBILITY_EXTENSION_LOADED") !== "true") {
+    console.log(`Not a Accessibility Automation session, cannot retrieve Accessibility results.`);
+    return
+  }
+  return new Promise(function (resolve, reject) {
+    try{
+      const e = new CustomEvent('A11Y_TAP_GET_RESULTS_SUMMARY');
+      const fn = function (event) {
+        window.parent.removeEventListener('A11Y_RESULTS_SUMMARY_RESPONSE', fn);
+          resolve(event.detail.summary);
+      };
+      window.parent.addEventListener('A11Y_RESULTS_SUMMARY_RESPONSE', fn);
+      window.parent.dispatchEvent(e);
+    } catch (err) {
+      console.log("No accessibility results summary was found.");
+      reject(err);
+    }
+  });
+});
+
+Cypress.Commands.add('getAccessibilityResults', () => {
+  if (Cypress.env("IS_ACCESSIBILITY_EXTENSION_LOADED") !== "true") {
+    console.log(`Not a Accessibility Automation session, cannot retrieve Accessibility results.`);
+    return 
+  }
+  return new Promise(function (resolve, reject) {
+    try{
+      const e = new CustomEvent('A11Y_TAP_GET_RESULTS');
+      const fn = function (event) {
+        window.parent.removeEventListener('A11Y_RESULTS_RESPONSE', fn);
+          resolve(event.detail.summary);
+      };
+      window.parent.addEventListener('A11Y_RESULTS_RESPONSE', fn);
+      window.parent.dispatchEvent(e);
+    } catch (err) {
+      console.log("No accessibility results were found.");
```

Comment:
Let's remove all the `console.log`

Reply:
These logs are printed in automate dashboard, hence not using logger.debug